wallet/etc/deploy/deploy-site.sh

```diff
@@ -3,14 +3,14 @@ set -e
 set -u
 
 if [ "${STAGE}" == "prod" ]; then
-  DISTRIBUTION=E2S4Y0ZV7EAMWM
+  DISTRIBUTION=E26KAJ3X9S922B
   BUCKET=cryptid.identity.com
 elif [ ${STAGE} == "preprod" ]; then
   DISTRIBUTION=???
   BUCKET=explorer-preprod.identity.com
 elif [ ${STAGE} == "dev" ]; then
-  DISTRIBUTION=E1SU4WO39BJJ5E
+  DISTRIBUTION=E2JAYNP6V36M37
   BUCKET=cryptid-dev.identity.com
 fi
 
-npx deploy-aws-s3-cloudfront --non-interactive --react --bucket ${BUCKET} --destination ${STAGE} --distribution ${DISTRIBUTION}
```

Comment:
I removed the destination (folder) in favour of a seperate bucket - let me know if you disagree